app/views/user/backup_code_mailer/backup_codes_enabled.html.erb

```diff
@@ -0,0 +1,16 @@
+<p>
+  Hey <%= @user.first_name %>,
+</p>
+
+<p>
+  You've just enabled backup codes for HCB, and have 10 codes available to use if you lose access to other means of accessing your account. Please make sure to save the codes in a safe place. HCB staff will never ask for your backup codes.
```

Comment:
```suggestion
  You've just enabled backup codes for HCB and have 10 codes available. Please make sure to save the codes in a safe place in case you lose access to your account. HCB staff will never ask for your backup codes.
```